### ISSUE TITLE
Hardware: alternative case proposal

### ISSUE THREAD

**Issue by dc1rdb** [sharebrained/portapack-hackrf#50]
Alternative case, based on the original HackRF plastic enclosure. The bottom half of the case could be reused. Just the top half of the plastic enclosure would need to be replaced by a version with extended side walls and cut-outs at the top for display and navigation control, plus additional hole for headphone jack and slot for SD-card.

**Comment by jboone** [comment]
@dc1rdb That's a great idea. However, I'm not much of a fan of plastic cases, and the cost of getting injection molding set up is vastly higher than I can afford.

But with 3D printers everywhere, perhaps a printable model could be made, based on your idea. If somebody wants to tackle that before I get to it, I highly recommend both OpenSCAD and FreeCAD. I have OpenSCAD models of the PortaPack and HackRF boards in the repo already that could be used to validate the positions of holes, edges, etc. I'm not sure if I posted my FreeCAD files yet...

**Comment by KC5CQW** [comment]
Sadly, the nature of this hardware [almost certainly] requires a metallic case for EMI mitigation. Such is the nature of RF.
It would be possible to print a case then line it with foil tape as an alternative with "limited" success.

For those not critical of EMI, or as a stop-gap measure, a printable case is a great idea to protect this valuable hardware. Personally, I would be willing to act as a beta tester and print one out if provided an .stl file. I am unable to commit to drafting a model at this time due to other obligations.

**Comment by ne1** [comment]
I have done a case, but i have to redesign it a little bit, to make it fits better. I will upload it on monday. 
I can send these to somebody, if anyone needs it.

**Comment by ne1** [comment]
Hi, sorry it takes a little longer and there is still room to improve this model, here is one of my test cases:

![img_3854](https://cloud.githubusercontent.com/assets/13911116/9478676/7da602e2-4b79-11e5-8a00-7e0865214604.JPG)
![img_3853](https://cloud.githubusercontent.com/assets/13911116/9478681/8474de7c-4b79-11e5-936a-dec65e4ad65c.JPG)
![img_3855](https://cloud.githubusercontent.com/assets/13911116/9478684/88d20d6e-4b79-11e5-859f-95afadf0a06b.JPG)

You can get it here: 
http://www.thingiverse.com/thing:983995

**Comment by dc1rdb** [comment]
This is looking great! Regarding KC5CQWs concerns about RF shielding, this could be addressed by covering the inside with conductive graphite spray paint.

**Comment by ne1** [comment]
Thanks! If anybody interested: i can print and send it for just the shipping costs :) (sitting in Germany) 

This spray should not touch the bottom side of the hackrf, correct? This could be possible in the actual design. 
But i can give a try. Is aluminium foil a suitable alternative?

**Comment by KC5CQW** [comment]
Add 2.35mm between the HackRF's button and SMA holes and the PortaPack's face to the case design to allow for 14.25mm spacers when a HackRF Blue or HackRF with the P30 header is installed. Generate a second set of STL's for this optional configuration.

**Comment by ablanquart** [comment]
If enough people are interested, I may be able to have this made in metal. I'll see if I can run a "sample" using ne1's thingiverse file.

**Comment by ne1** [comment]
I can send you my FreeCad File if you like!?

**Comment by ablanquart** [comment]
Absolutely, please send! Not sure if you can convert or make easily - but a
.dxf or a .dwg file would be better.  I think I can work with the STL and
FreeCAD files though.

On Sun, Jan 24, 2016 at 2:16 AM, ne1 <notifications@github.com> wrote:

> I can send you my FreeCad File if you like!?
>
>
> > Am 24.01.2016 um 03:44 schrieb ablanquart <notifications@github.com>:
> >
> > If enough people are interested, I may be able to have this made in
> metal. I'll see if I can run a "sample" using ne1's thingiverse file.
> >
> > —
> > Reply to this email directly or view it on GitHub <
> https://github.com/sharebrained/portapack-hackrf/issues/50#issuecomment-174244649
> >.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sharebrained/portapack-hackrf/issues/50#issuecomment-174276837>
> .
>



-- 
ADAM BLANQUART | ablanquart@gmail.com |

**Comment by ne1** [comment]
Hi,

here is the file - i hope it can help. 





> Am 26.01.2016 um 17:10 schrieb ablanquart <notifications@github.com>:
> 
> Absolutely, please send! Not sure if you can convert or make easily - but a
> .dxf or a .dwg file would be better. I think I can work with the STL and
> FreeCAD files though.
>
